### PR TITLE
Autoscaling for candle and view

### PR DESCRIPTION
**Pull request by buzzCraft**
As requested in #180 I added a way to autoscale all plots. Implemented the changes in candle and view, and can easy be implemented to all plots. 

Added screeninfo to requirements.txt
Screeninfo should work cross platform, but i'v only tested it on Windows 10

### CODE REVIEW

**Comment by buzzCraft** [comment]
Getting an error when importing screeninfo. Iv included it into:
requirements.txt
requirements-full.txt
requirements-win.txt
not sure where i went wrong (unit testing is a new field for me)

**Comment by DidierRLopes** [comment]
I think you need to update the poetry lock file.

Can you do: "poetry add screeninfo" and then test with "poetry build". This should update both pyproject.toml and poetry.lock.

**Comment by buzzCraft** [comment]
Ok, that seems to work, please check poetry.lock, as I don't understand all changes that happen there.
If this scaling feature is something you want to use, i'll update all the plots later.

File: gamestonk_terminal/config_plot.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+SCREEN_HEIGHT = 0
+SCREEN_WIDTH = 0
```
Comment (DidierRLopes) [inline gamestonk_terminal/config_plot.py:2]: I think those values are never used, as screeninfo library is getting it for us

File: gamestonk_terminal/config_plot.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+PLOT_HEIGHT_PRECENTAGE = 100
+PLOT_WIDTH_PRECENTAGE = 50
+
+PLOT_HEIGHT = 800
+PLOT_WIDTH = 800
```
Comment (DidierRLopes) [inline gamestonk_terminal/config_plot.py:8]: Also, the defaults look like this to me:

USE_PLOT_AUTOSCALING = True
Captura de ecrã 2021-03-16, às 19 22 03

USE_PLOT_AUTOSCALING = False
Captura de ecrã 2021-03-16, às 19 19 29

May I suggest these:
```
PLOT_HEIGHT_PRECENTAGE = 50
PLOT_WIDTH_PRECENTAGE = 70

PLOT_HEIGHT = 500
PLOT_WIDTH = 800
```

Comment (DidierRLopes) [reply gamestonk_terminal/config_plot.py:8]: Also can you add a comment above these parameters that:
PLOT_HEIGHT_PRECENTAGE
PLOT_WIDTH_PRECENTAGE
are used when AUTO_PLOT is set to True,

And that the others are used when AUTO_PLOT is False.

Comment (DidierRLopes) [reply gamestonk_terminal/config_plot.py:8]: Precentage is mispelled, it should be "Percentage"

File: gamestonk_terminal/helper_funcs.py [hunk 3]
```diff
@@ -353,3 +355,25 @@ def get_flair() -> str:
         return flair[gtff.USE_FLAIR]
 
     return ""
+
+
+def get_screeninfo():
+    screens = get_monitors()  # Get all available monitors
+    main_screen = screens[0]  # Choose the first screen as main screen
```
Comment (DidierRLopes) [inline gamestonk_terminal/helper_funcs.py:362]: Would be nice having the MAIN/SECONDARY screen as a configurable parameter!

**Review by DidierRLopes — CHANGES_REQUESTED** [review]
Some minor tings because I'm picky, but overall I just tested this and it is great!! Feel free to apply it to the TA and PRED figures please. #welikeniceplots

File: gamestonk_terminal/config_plot.py (hunk 2 above)
Comment (buzzCraft) [reply gamestonk_terminal/config_plot.py:8]: Thanks for the reply. I'll get right to it:)
I really appreciate the feedback, as I'm new to coding in teams, and got no formal / educational coding background.

Comment (DidierRLopes) [reply gamestonk_terminal/config_plot.py:8]: For someone with no coding background you're doing amazing!! I'll wait for your changes to merge this :) Thank you!

**Comment by buzzCraft** [comment]
Okay, think I covered all of your request. I also added scaling to momentum, overlap, trend_indicators, volatility and volume. At of this moment I don't have my env ready for predictions, so I didn't want to start adding scaling there without the possibility to test it. 
I'll get predictions up and running as soon as possible (within a day or two).

**Review by DidierRLopes — APPROVED** [review]

**Comment by DidierRLopes** [comment]
Again, this is amazing. Great contribution. Thank you for this :)

**Pull request merged**